.github/workflows/tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  unittests-linux:
+    strategy:
+      matrix:
+        python-version:
+          - "3.11"
+        runner: ["linux.12xlarge"]
+        gpu-arch-type: ["cpu"]
+        include:
+          - python-version: 3.8
+            runner: linux.g5.4xlarge.nvidia.gpu
+            gpu-arch-type: cuda
+            gpu-arch-version: "11.8"
```

Comment:
maybe 12.1?

Reply:
I'm not sure this is possible, it depends on what github runners are offered to the pytorch org